### PR TITLE
Define kernel variables on phpunit xml dist

### PR DESCRIPTION
**Pull request by jordisala1991**

### CODE REVIEW

File: templates/project/phpunit.xml.dist.twig [hunk 1]
```diff
@@ -48,6 +48,11 @@ It's auto-generated by sonata-project/dev-kit package.
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
 {% if hasPanther %}
         <server name="PANTHER_WEB_SERVER_DIR" value="./tests/App/public/" />
+{% endif %}
+{% if file_exists(project_dir ~ '/tests/App/AppKernel.php') %}
+        <env name="KERNEL_CLASS" value="\Sonata\PageBundle\Tests\App\AppKernel" />
```
Comment (jordisala1991) [inline templates/project/phpunit.xml.dist.twig:53]:
```suggestion
        <env name="KERNEL_CLASS" value="\Sonata\{{ project.namespace }}\Tests\App\AppKernel" />
```

**Comment by jordisala1991** [comment]
I am not sure if I used correctly the new function @VincentLanglet 

Also maybe we should wait until I get somewhere near a laptop, working with ipad is not ideal :p

**Comment by VincentLanglet** [comment]
> I am not sure if I used correctly the new function @VincentLanglet
> 
> Also maybe we should wait until I get somewhere near a laptop, working with ipad is not ideal :p

Yeah, dont hurry we can wait for a laptop.

You can test the dispatch-files locally (it will be a dry-run mode) to see the diff

**Comment by jordisala1991** [comment]
I tried the branch and it works correctly. I have time now to do all the fixes on all branches.

**Pull request merged**